src/__tests__/orchestrator.integration.spec.ts

```diff
@@ -69,15 +69,18 @@ describe('elastic orchestrator', () => {
 
     let client: Client
 
-    const createBulkPayload = (count: number) =>
+    const createBulkPayload = (
+      count: number,
+      bodyGenerator: (value: number) => object = value => ({
```

Comment:
Do you think we can extract this type into a type variable? I'd make the code more readable.

Reply:
👍